test/workspace.js

```diff
@@ -249,4 +249,61 @@ describe('Workspace', function() {
       });
     });
   });
+
+  describe('Creating 2 projects from template', function() {
```

Comment:
Did you mean to move the test cases into the previous describe block above?
https://github.com/strongloop/loopback-workspace/blob/master/test/workspace.js#L100